move/interchain_token_service/sources/versioned/interchain_token_service_v0.move

```diff
@@ -608,35 +592,29 @@ fun add_registered_coin<T>(
     );
 }
 
-/// Send a payload to a destination chain. The destination chain needs to have a
-/// trusted address.
-fun prepare_message(
+fun prepare_wrapped_message(
```

Comment:
The ITS spec distinguishes between hub and regular ITS message, so I'd prefer using the same terminology of `hub` instead of `wrapped`. Alternatively, call this `prepare_message`, and rename prepare_message to `prepare_gateway_message`
```suggestion
fun prepare_hub_message(
```